dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -56,6 +56,10 @@ namespaces:
         type: timer
         tags: [ channel-name ]
         docs: Time spent waiting in the sticky queue before execution attempt.
+      requests.size:
+        type: histogram
+        tags: [service-name, endpoint]
```

Comment:
Let's include `channel-name` which is roughly equivalent to the service name in a service discovery block (this shouldn't increase cardinality)